apps/src/code-studio/pd/workshop_dashboard/components/workshop_form.jsx

```diff
@@ -891,6 +900,10 @@ export class WorkshopForm extends React.Component {
     return !['Counselor', 'Admin'].includes(this.state.course);
   }
 
+  shouldRenderModules() {
+    return this.state.subject === 'Custom Workshop';
```

Comment:
Nit: if possible, could this reference `SUBJECT_CUSTOM_WORKSHOP` instead of `Custom Workshop` in case we change the wording in the future?